### ISSUE TITLE
Question around cross platform images.

### ISSUE THREAD

**Issue by gnikolov95** [cekit/cekit#922]
**Describe the solution you'd like**
I've seen awhile ago it's been added support for multi platform e.g `--platfrom linux/amd64,linux/arm64`
I've tried this and I might be ignorant here but I can see the two images being built - one for arm64 one for amd64 but they'd produce a single image in the end which is not a manifest, upon inspecting it I have come to realise that it is only `amd64` and the `arm64` gets lost.

Just for the sake of it I've pushed the image to a registry and tried to pull it back even though I could clearly see single manifest and it come's back as x86_64 not an aarch64 as the laptop I was testing from.

Could you elaborate on this a bit, I could build each of those one by one then create a manifest and push that but, I am a bit lost what the supposed multi platform support actually achieve, I also understand I could be totally ignorant about cekit because I've started using it just recently.

Since I am building with podman, usually there would be a `--manifest` flag that could be passed but it doesn't appear to be supported by cekit. 

**Additional context**
Log of the command that I've run and the outputs with -v flag of it. cekit - 4.13.0
[log.txt](https://github.com/user-attachments/files/17425088/log.txt)

**Comment by rnc** [comment]
Relevant issue : #775 
You are correct that the manifest flag isn't supported and no work has been for multiarch support for podman.

**Comment by rnc** [comment]
There will be the ability to pass arbitrary flags such as `--manifest` ; will this satisfy your requirements?

**Comment by gnikolov95** [comment]
@rnc Yes this would be great, I had a look at the PR and it covers the use case I was mentioning.
